provider/datagen/src/transform/segmenter/mod.rs

```diff
@@ -333,8 +333,9 @@ impl SegmenterRuleProvider {
         // The table length should be large enough to contain all codepoints.
         const UAX29_CODEPOINT_TABLE_LEN: usize = 0xE1000;
 
-        // The property values of codepoints >= U+0x20000 are built into the line segmenter.
-        const UAX14_CODEPOINT_TABLE_LEN: usize = 0x20000;
+        // The property values of codepoints >= U+0x20000 could be built into the line segmenter,
+        // but we instead store them compactly in the CodePointTrie.
+        const UAX14_CODEPOINT_TABLE_LEN: usize = 0xE1000;
```

Comment:
UAX29's table size and UAX14 table size are same now, so it is unnecessary to define two value.